src/Uno.UWP/Devices/Geolocation/Geolocator.iOSmacOS.cs

```diff
@@ -21,10 +21,6 @@ public Geolocator()
 				DesiredAccuracy = DesiredAccuracy == PositionAccuracy.Default ? 10 : 1,
 			};
 
-#if __IOS__ //required only for iOS

```

Comment:
@MartinZikmund do you remember why you had to add this here ?

Reply:
Originally added to support backgrounded scenarios, but we shouldn't force it on Geolocator users unless explicitly needed. Without this call, the Geolocator will still work properly when in the foreground. We can remove this for now and maybe we will have to add a platform-specific method to allow users to request background support later.